scripts/release

```diff
@@ -307,8 +308,10 @@ merge_release() {
     git checkout develop
   fi
   git rebase origin/develop
-  git merge --no-ff release-candidate --no-edit
-
+  git merge --no-ff --no-commit release-candidate --no-edit
```

Comment:
Suggested comment for clarity (because it wasn't clear to me that this results in one commit:

```bash
git rebase origin/develop

# The following lines will only create one commit thanks to the --no-commit argument here.
git merge --no-ff --no-commit release-candidate --no-edit
git reset HEAD .gitattributes
git checkout -- .gitattributes
git commit -m "merged release-candidate"

git branch -D release-candidate
```

Reply:
Done, thanks for the feedback